templates/framework/dashboard.html

```diff
@@ -113,6 +113,9 @@ <h1 class="page-header">My Posts</h1>
                     <div class="row col-md-10" style="margin: 0; padding: 0;">
                         {% include 'posts/all.html' %}
                     </div>
+		    <div class="row col-md-10" style="margin: 0; padding: 0;">
+                        {% include 'posts/github_stream.html' %}
```

Comment:
Once it is changed to github_feeds I can load the dashbaord agian, but I am not getting any GitHub feed at all to my dashbaord. I'm using Admin which is linked to grepme.

This may be close, but I don't think it is ready to be merged yet.